cvat-ui/src/components/create-job-page/job-form.tsx

```diff
@@ -86,24 +90,40 @@ function JobForm(props: Props): JSX.Element {
         }
     };
 
+    const sizeBase = useCallback(() => {
+        if (frameSelectionMethod === FrameSelectionMethod.RANDOM) {
+            return taskSize;
+        }
+        return segmentSize;
+    }, [frameSelectionMethod, segmentSize, taskSize]);
```

Comment:
I do not think we need `useCallback` here as this function is not used as a prop somewhere